package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "prettierlichess",
```

Comment:
Why did you delete the name here?

Reply:
I really don't know why this line deleted